src/components/rmrk/Gallery/GalleryCard.vue

```diff
@@ -5,7 +5,7 @@
         <b-image
           :src="image"
           :src-fallback="placeholder"
-          alt="Simple image"
+          :alt="name || 'Simple image'"
```

Comment:
@roiLeo you use `name` here instead of `title`

Reply:
Oh nice catch